### ISSUE TITLE
createPracticeLobby with certain settings failed

### ISSUE THREAD

**Issue by gerlaic** [Arcana/node-dota2#432]
I uncommented the code in example with some little changes:
```
Dota2.createPracticeLobby("password",
                    {"game_name": "node-dota2",
                    "server_region": 1,
                    "game_mode": 1,
                    "fill_with_bots": true,
                    "allow_spectating": true,
                    "pass_key": "password"
                    },
                    function(err, body){
                         console.log(JSON.stringify(body));
                         Dota2.inviteToLobby('24923508229995526');
                         Dota2.joinPracticeLobbyTeam(2,4);
                         Dota2.on('practiceLobbyUpdate', function(){
                           console.log(arguments);
                         });
                    });
```
The game_mode in both client and arguments from practiceLobbyUpdate is 5. And the attribute "fill_with_bots" is false. 
```
{ '0':
   CSODOTALobby {
     lobby_id: Long { low: 1989868550, high: 5802956, unsigned: true },
     members: [ [Object] ],
     game_mode: 5,
     state: 0,
     leader_id: Long { low: 360059900, high: 17825793, unsigned: true },
     lobby_type: 1,
     allow_cheats: false,
     fill_with_bots: false,
     intro_mode: false,
     game_name: 'node-dota2',
     server_region: 0,
     cm_pick: 0,
     allow_spectating: true,
     bot_difficulty_radiant: 0,
     game_version: 1,
     pass_key: 'password',
     leagueid: 0,
     penalty_level_radiant: 0,
     penalty_level_dire: 0,
     series_type: 2,
     radiant_series_wins: 0,
     dire_series_wins: 0,
     allchat: true,
     dota_tv_delay: 1,
     lan: false,
     broadcast_channel_info: [ [Object] ],
     lan_host_ping_to_server_region: 0,
     visibility: 0,
     league_series_id: 0,
     league_game_id: 0,
     previous_match_override: Long { low: 0, high: 0, unsigned: true },
     league_selection_priority_team: 0,
     league_selection_priority_choice: 0,
     league_non_selection_priority_choice: 0,
     pause_setting: 0,
     bot_difficulty_dire: 0,
     bot_radiant: Long { low: 0, high: 0, unsigned: true },
     bot_dire: Long { low: 0, high: 0, unsigned: true } } }
```

What happened here?

**Comment by devilesk** [comment]
Are you sure that the account isn't in any existing lobbies before trying to create that new lobby?

In my own tests my account got stuck in an invisible lobby (not sure how though). But I got the bot code to work after manually logging into the account and disconnecting it from the lobby it was in and then trying again.

To leave the lobby, I had to do a couple of things I found in this [reddit thread](https://www.reddit.com/r/DotA2/comments/4jef2m/im_stuck_in_a_hidden_lobby_i_cant_leave/). I entered `map dota` into the console, it launched a local game, I chose a side and hero and then while I was in the game I opened the console again and entered `force_leave_game_button 1`. Then when I went back to the dashboard there was the red Leave Game button above the disconnect button. Clicking that and then disconnect got me out of the invisible lobby I was in.
